8/community/Dockerfile

```
@@ -86,7 +86,7 @@ RUN set -eux; \
 #
 # SonarQube setup
 #
-ARG SONARQUBE_VERSION=8.9.6.50800
+ARG SONARQUBE_VERSION=8.9.7.50800
```

The `SONARQUBE_VERSION` is used to download the SQ zip file from Binaries.

Could you change it back to this?
```suggestion
ARG SONARQUBE_VERSION=8.9.6.50800
```

This should help on making the checks go green